sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuParquetScan.scala

```diff
@@ -2611,8 +2581,14 @@ object MakeParquetTableProducer extends Logging {
       debugDumpAlways: Boolean
   ): GpuDataProducer[Table] = {
     if (useChunkedReader) {
-      ParquetTableReader(conf, chunkSizeByteLimit, opts, buffer, offset, len, metrics,
-        dateRebaseMode, timestampRebaseMode, hasInt96Timestamps,
+      val passReadLimit = if (useSubPageChunked) {
+        4 * chunkSizeByteLimit
```

Comment:
Can we set some multiplier constant or configurable constant for it, instead of hard coding `4X` like this?

Reply:
I'm not sure where would be a good place to put this. It is a magic number based on our estimates that we will take 4x the target batch size as the amount of memory that we are allowed to use. GpuDataProducer is the only place that might be common between them, but that is not a proper place for it. I could create a static object to hold it somewhere. GpuConventionMagicNumbers or something.

Reply:
Okay then let's see what can we do the next time with the ORC chunked reader. I'm fine to leave this as-is for now.